server/util.go

```diff
@@ -105,7 +105,9 @@ func interceptLoadConfig() (conf *cfg.Config, err error) {
 		conf, err = tcmd.ParseConfig() // NOTE: ParseConfig() creates dir/files as necessary.
 	}
 
-	// create a default gaia config file if it does not exist
+	// create a default Gaia config file if it does not exist
+	//
+	// TODO: Rename config file to server.toml as it's not particular to Gaia
```

Comment:
We might want to open an issue here...

Reply:
I think this is a `tendermint` issue maybe? Cause I can imagine that many apps might want to have application specific configuration as well as the standard `tendermint` configuration.

Reply:
Agreed, opened an issue here: https://github.com/cosmos/cosmos-sdk/issues/4125

WRT to it being a Tendermint issue, I don't see how this relates to TM? This is really a BaseApp configuration. Tendermint config exists in `config.toml` and the node/server/baseapp config currently live in `gaiad.toml` but it really has nothing to do with gaia.